_layouts/base.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="{{ page.lang | default: site.lang | default: 'en-US' }}">
+  <head>
+    {% include meta.html %}
+    {% include styles.html %}
+    {% include google_analytics.html %}
+    {% include dap.html %}
+    {% include google_recaptcha.html %}
+  </head>
+  <body
+    class="{{ layout.class }} {{ page.class }} site lang-{{ site.lang }}"
```

Comment:
Noting that the 404 page now includes `site` and `lang-` classes. The `site` class appears to have some associated flex styling, but since there's only a single element child of the body, it doesn't affect anything.

tl;dr: Fine 👍